cli/server.go

```diff
@@ -53,6 +53,8 @@ import (
 	"gopkg.in/yaml.v3"
 	"tailscale.com/tailcfg"
 
+	"github.com/coder/coder/v2/coderd/database/dbauthz"
+
```

Comment:
nit: move with the rest of the coder/coder imports

Reply:
Strange how the formatter doesn't do this for us; will do 👍